pages/common/gum.md

```diff
@@ -0,0 +1,28 @@
+# gum
+
+> A tool for making glamorous shell scripts.
+> More information: <https://github.com/charmbracelet/gum>.
+
+- Interactively pick an option to print to `STDOUT`:
+
+`gum choose "{{option_1}}" "{{option_2}}" "{{option_3}}"`
```

Comment:
```suggestion
`gum choose "{{option1}}" "{{option2}}" "{{option3}}"`
```

Reply:
There can also be as many options, can we include a ...

Reply:
This is incorrect syntax, 

the each option must be encased with quotation marks.

What do you think of

```suggestion
`gum choose "{{option_1}}" "{{option_2}}" "{{option_3}}" "{{...}}"`
```

Reply:
> the each option must be encased with quotation marks.

But it is [valid](https://github.com/tldr-pages/tldr/blob/main/pages/linux/zip.md) for directories:

```
{{path/to/directory1 path/to/directory2 ...}}
```

🤔

Reply:
$ gum choose "one two three"
> one two three

This is not what we want. We want it to be
> One 
> Two
> Three

since the current output gives one "option"

Reply:
We can put quotes inside placeholder.

Reply:
Let me watch this .. I am the most confused person when it comes to `placeholder` syntax :)
👀

Reply:
@EmilySeville7cfg we always put quotes outside of the placeholders. Because the quote itself is not part of the argument, it is always required. In all our English pages, we have almost 700 matches for `"{{...}}"` and only 7 for `{{"..."}}` (which we should fix).

Reply:
Got it.